src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,24 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
     /**
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
-     * @param password - user's password
+     * @param emailEntered - user's email
+     * @param passwordEntered - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
-    public boolean login(String email, String password) {
-        return false;
+    public boolean login(String emailEntered, String passwordEntered) {
+        UserService userService = new UserService();
+        User findedUser = userService.findByEmail(emailEntered);
+        if (findedUser == null) {
+            return false;
+        } else {
+            return passwordEntered.equals(findedUser.getPassword());
+        }
```

Comment:
```suggestion
        } 
        return passwordEntered.equals(findedUser.getPassword());
```
we don't need else after return

Reply:
Thanks for advice. I fixed it